src/mixs/schema/ancient.yml

```diff
@@ -83,13 +90,15 @@ slots:
     range: Ancient
     inlined: true
     inlined_as_list: true
-  collection_context_name:
+  source_mat_curat_insti:
     annotations:
-      Expected_value: name of institution
-    description: Name of location where sample is typically stored or archived. Typically will be 'owning institution'.
-    title: location where sample typically is archived
```

Comment:
See comment below ('source mateiral)'

Reply:
Done in https://github.com/MIxS-MInAS/extension-ancient/pull/14/commits/1e319e351557d2a4c0b25691635b2f39e1d92383